lib/std/paths.nim

```diff
@@ -39,7 +39,7 @@ func add(x: var string, tail: string) =
 
 func add*(x: var Path, y: Path) {.borrow.}
 
-func `/`*(head, tail: Path): Path {.inline.} =
+func `/`*(head, tail: Path | string{lit}): Path {.inline.} =
```

Comment:
```suggestion
func `/`*(head Path; tail: static[string]): Path {.inline.} =
```

But that doesn't mean the RFC is accepted.

Reply:
oh, right. should i close the pr?

Reply:
Semicolon is preferred yes.

Reply:
I personally find comma one more readable. anyways, will follow current files's standard of parameter convention (commas)